_includes/scenarios/integrate/identityattribute-introduction.md

```diff
@@ -0,0 +1,23 @@
+An [Identity Attribute]({% link _docs_integrate/attribute-values.md %}#identity-attributes) is a specific piece of information associated with an identity (person or entity) in the context of the networked ecosystem. Attribute values can take on different types that define the format, validation rules, and presentation information for the stored data.
```

Comment:
- To be consistent with other documentation pages, I would write "IdentityAttribute" instead of "Identity Attribute" (also in other places below) and "Identity" instead of "identity"
- I'd link here ("[Identity Attribute]({% link _docs_integrate/attribute-values.md %}#identity-attributes)") to the IdentityAttribute in the Data Model Overview instead and not yet to the IdentityAttribute value type page. I'd replace in the second sentence "Attribute values" with  "[Attribute values]({% link _docs_integrate/attribute-values.md %}#identity-attributes)" then